src/modules/files/controllers/file.controller.ts

```diff
@@ -0,0 +1,69 @@
+import { Body, Controller, Delete, HttpCode, Param, ParseUUIDPipe, Post, Req, Res } from '@nestjs/common'
+import { ApiResponse, ApiTags } from '@nestjs/swagger'
+import { Response } from 'express'
+import { Request } from '../../auth/guards/auth.guard.js'
+import { CreateFileDto } from '../dtos/create-file.dto.js'
+import { FileCreatedTransformerType, FileCreatedTransformer } from '../transformers/file-created.transformer.js'
+import { FileService } from '../services/file.service.js'
+
+@ApiTags('File')
+@Controller('file')
+export class FileController {
+  constructor (private readonly fileService: FileService) {}
+  @Post()
```

Comment:
lege lijn tussen vlammen

Reply:
vind dit ook mooier precies 🙊 
```suggestion
  constructor (
    private readonly fileService: FileService
  ) {}
  
  @Post()
```